blackjack/lib/blackjack/dealer.ex

```diff
@@ -0,0 +1,179 @@
+defmodule Blackjack.Dealer do
```

Comment:
This is really nice looking code: good naming and very well laid out.

My only thought is that it's kinda big: I wonder if it could have been split each time you have a separator comment.

(no deduction; just an observation)